amgut/lib/data_access/ag_data_access.py

```diff
@@ -944,7 +950,7 @@ def addParticipantException(self, ag_login_id, participant_name):
 
     def handoutCheck(self, username, password):
         cursor = self.get_cursor()
-        cursor.execute("""SELECT distinct(password)
+        cursor.execute("""SELECT DISTINCT (password)
```

Comment:
I *think* the new ag_handout_kits table will have only one row per kit ID, which means only one password, which means that the `distinct` here should be unnecessary. If all of my assumptions are correct and things break when the `distinct` keyword is removed, it's probably a cause for concern.